app/services/manual_payout_report_publisher_includer.rb

```diff
@@ -0,0 +1,42 @@
+class ManualPayoutReportPublisherIncluder < BaseService
```

Comment:
@corymcdonald and I have thoughts that we should change the naming conventions that we currently have. Definitely worth it for us to have the discussion and make sure naming conventions are conventional and obvious.

I think this class should be `PayoutReport::ManualGenerationService` or `PayoutReport::ManualGeneration`. 

cc: @nvonpentz who will also have thoughts too.

Reply:
I used naming conventions similar to the existing codebase 

thus `manual_payout_report_publisher_includer.rb` is the manual complement of the existing class `payout_report_publisher_includer.rb`

Reply:
I like adding more name spacing when we can like you have described @yachtcaptain23, but would advocate for keeping the syntax of the services that since that would keep everything consistent and because I don't see the downsides of the current method (please point them out).